gateway-messages/src/lib.rs

```diff
@@ -141,6 +141,35 @@ impl From<UpdateId> for uuid::Uuid {
     }
 }
 
+/// Affect boot image selection after reset.
+#[derive(
+    Debug,
+    Clone,
+    Copy,
+    PartialEq,
+    Eq,
+    Hash,
+    SerializedSize,
+    Serialize,
+    Deserialize,
+)]
+pub enum ResetIntent {
+    /// Perform a normal reset without altering boot image selection.
+    Normal = 1,
```

Comment:
Do we need explicit values on this and `ExpensiveAndIrrevocableProdToDev`? They aren't used by hubpack when serializing so seem a little misleading.

Reply:
That "=1" can go away.